Extensions/xkit_preferences.js

```diff
@@ -734,6 +741,9 @@ XKit.extensions.xkit_preferences = new Object({
 			$("#xkit-control-panel").remove();
 		});
 
+		// Remove esc key listener attached on open
+		$(document).off('keydown');
```

Comment:
This removes every keydown listener attached to the document. Tumblr's code attaches a few keydown listeners (though I don't know what they use them for)

This should instead use the targeted version of `.off`, which requires making the keydown listener a property of xkit_preferences. Something like

```javascript
on_modal_keydown: function(event) {
// ....
},
```

would work properly and could be referenced in `on` and `off` as `XKit.extensions.xkit_preferences.on_modal_keydown`.